modules/extensions/activeworker.tf

```
@@ -27,8 +27,7 @@ resource "null_resource" "check_worker_active" {
 
   provisioner "remote-exec" {
     inline = [
-      "chmod +x $HOME/check_active_worker.sh",
-      "$HOME/check_active_worker.sh"
+      "if [ -f \"$HOME/check_active_worker.sh\" ]; then bash $HOME/check_active_worker.sh; rm -f \"$HOME/check_active_worker.sh\";fi",
```

```suggestion
      'if [ -f "$HOME/check_active_worker.sh" ]; then bash "$HOME/check_active_worker.sh" && rm -f "$HOME/check_active_worker.sh"; fi',
```

Though, I'm not sure if HCL allows for single quoted strings in lists to preserve double quotes. Hopefully it does. If so, this pattern can be repeated for the rest, as we really do want to wrap the path to the script, if possible.